cpplint_unittest.py

```diff
@@ -4282,8 +4282,8 @@ def testParseArguments(self):
       self.assertEqual(set(['hpp', 'h', 'cpp']), cpplint.GetAllExtensions())
 
     finally:
-      sys.stdout == sys.__stdout__
-      sys.stderr == sys.__stderr__
+      sys.stdout = sys.__stdout__
+      sys.stderr = sys.__stderr__
```

Comment:
Careful review, please.  The current code is a useless compare.  The proposed code restores the old value.

Reply:
I would think restoring is what it meant indeed.